src/blp/blp.py

```diff
@@ -803,6 +800,44 @@ def collect_to_bdh(self, responses: Iterable) -> Dict[str, pandas.DataFrame]:
 
         return dfs
 
+    def beqs(
+        self,
+        screen_name: str,
+        screen_type: str = "PRIVATE",
+        overrides: Optional[Sequence] = None,
+        options: Optional[Dict] = None,
+    ) -> pandas.DataFrame:
+        """Bloomberg equity screening request.
+
+        Args:
+            screen_name: name of the screen
+            screen_type: type of screen, either 'PRIVATE' or 'GLOBAL'
+            overrides: List of tuples containing the field to override and its value
+            options: key value pairs to to set in request
+
+        Returns: A pandas.DataFrame with columns ['security', eqs_data[0], ...]
+        """
+        query = create_eqs_query(screen_name, screen_type, overrides, options)
+        df = self.query(query, self.parser, self.collect_to_beqs)
+        columns = ["security"] + sorted([col for col in df.columns if col != "security"])
+        df = (
+            df.sort_values("security").reset_index(drop=True).loc[:, columns].drop(["Ticker"], axis=1)
```

Comment:
What's the reason for dropping `Ticker`? Is this a redundant piece of information that is returned by the Bloomberg api that is always equal to `security`?

Reply:
Exactly! However, if you prefer, we could just leave the Ticker information and let the user drop it if he wants to.

Reply:
Ya I think that would be more consistent with other approaches in the library, where I try to pass through the Bloomberg response in a fairly unmodified way.